_gsocproposals/2019/proposal_AllenSIMTtoSPMD.md

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
Can you please remove this first empty line? Otherwise, the content is not generated.